Mage.Tests/src/test/java/org/mage/test/player/TestPlayer.java

```diff
@@ -4295,6 +4295,9 @@ public SpellAbility chooseAbilityForCast(Card card, Game game, boolean noMana) {
     @Override
     public SpellAbility chooseAbilityForCast(Card card, List<SpellAbility> spellAbilities, Game game) {
         assertAliasSupportInChoices(false);
+        if (spellAbilities.size() == 1) {
+            return spellAbilities.get(0);
+        }
```

Comment:
not sure if this should choose automatically if in strict mode but if there's only one option then it has to be chosen so it makes sense to me

Reply:
That's ok.